build-aux/tap-driver.sh

```diff
@@ -0,0 +1,651 @@
+#! /bin/sh
+# Copyright (C) 2011-2020 Free Software Foundation, Inc.
```

Comment:
```suggestion
# Copyright (C) 2011-2018 Free Software Foundation, Inc.
```
I think my system already has that script file installed and I got this diff. I am not sure if we want to apply this diff or rather put it into the gitignore file.

Reply:
Unfortunately adding it to the gitignore file won't work because the file is tracked (Git ignores .gitignore for tracked files).

This is one of the reasons I don't like to check in generated files. Whenever you run `autogen` you will see diffs like this unless the version of Automake you installed on your machine matches the version of Automake that provided this version of `tap-driver.sh`.

This file is checked in because Automake 1.11 and older doesn't come with it, and RHEL/CentOS 6 has Automake 1.11. RHEL/CentOS 6 reach end of life on November 30th, 2020, at which point we can bump the minimum required Automake version and delete this file from our repository.

Reply:
Git should ignore those changes if you commit the gitignore change.